src/SecurityManager.js

```diff
@@ -78,6 +88,22 @@ async function getSecretStorageKey({ keys: keyInfos }, ssssItemName) {
         return [keyId, secretStorageKeys[keyId]];
     }
 
+    // if we dehydrated a device, see if that key works for SSSS
+    if (dehydrationInfo.key) {
+        try {
+            if (await MatrixClientPeg.get().checkSecretStorageKey(dehydrationInfo.key, keyInfo)) {
```

Comment:
So, I am probably missing a key part of the entire plot here, but why is the dehydration key a (potential) 4S key...? Is that only true if you haven't setup 4S yourself with your own key / passphrase...?

Reply:
The idea is that users don't want to remember extra passphrases/keys, so we automatically use the same key for SSSS as we do for dehydration.